chapter_01.asciidoc

```diff
@@ -6,6 +6,8 @@ It's a discipline, like a martial art, and just like in a Kung Fu movie,
 you need a bad-tempered and unreasonable master to force you to learn the discipline.
 Ours is the Testing Goat.
 
+// JAN: As a reader, I have no idea why would I practice TDD in the first place. Why would I want to master this discipline? I think it would be worth mentioning here. Doesn't have to be very long.
+// Maybe something like: "TDD forces you to develop in small chunks. Therefore, you can focus on one thing at the time. As a byproduct, you have executable specification of your features in form of tests. Want to make sure your changes haven't broken any existing feature? Just run the tests. Fast and easy. And so is releasing and deployment when you really practice TDD - when you obey the testing goat"
```

Comment:
I guess I tried to cover some of the "why tdd" stuff in the [preface](https://github.com/hjwp/Book-TDD-Web-Dev-Python/blob/main/preface.asciidoc)

Reply:
I see. In that case, just ignore the comment 👍